src/Page/Form.purs

```diff
@@ -76,21 +78,24 @@ component =
     }
 
   render :: State -> H.ParentHTML Query (ChildQuery m) ChildSlot m
-  render { form } =
-    Layout.section_
-      case form of
-        NotAsked ->
-          []
-        Loading ->
-          [ Card.card_ [ Format.p [ css "text-red" ] [ HH.text "Loading..." ] ] ]
-        Failure e ->
-          [ Card.card_ [ Format.p [ css "text-red" ] [ renderEvalError e ] ] ]
-        Success page ->
-          append
-          [ Format.heading_
-            [ HH.text page.evaled.name ]
-          ]
-          $ renderSection <$> page.evaled.contents
+  render { form, values } =
+    HH.div_
+      [ Layout.section_
+        case form of
+          NotAsked ->
+            []
+          Loading ->
+            [ Card.card_ [ Format.p [ css "text-red" ] [ HH.text "Loading..." ] ] ]
+          Failure e ->
+            [ Card.card_ [ Format.p [ css "text-red" ] [ renderEvalError e ] ] ]
+          Success page ->
+            append
+            [ Format.heading_
+              [ HH.text page.evaled.name ]
+            ]
+            $ renderSection <$> page.evaled.contents
+      , HH.pre_ [ HH.text $ show values ]
```

Comment:
Making a comment here, so if it disappears feel free to ignore it (I'm going commit by commit). I assume this is for debugging, let's not forget to remove this.

Reply:
Conversation is in: https://github.com/citizennet/purescript-lynx/pull/16#discussion_r268311886.